projects/ngx-xapi/model/src/lib/statement.ts

```diff
@@ -24,7 +25,7 @@ export interface Context {
   contextActivities?: ContextActivities;
   revision?: string;
   platform?: string;
-  language?: string;
+  language?: Exclude<string, 'und' | 'Und' | 'UND'>;
```

Comment:
Is there a locale type?

Reply:
no

Reply:
What about this?

https://www.npmjs.com/package/@types/locale